typing/env.ml

```diff
@@ -3171,9 +3175,13 @@ let report_lookup_error _loc env ppf = function
       fprintf ppf "@[The functor %a is generative,@ it@ cannot@ be@ \
                    applied@ in@ type@ expressions@]" !print_longident lid
   | Cannot_scrape_alias(lid, p) ->
+      let cause =
+        if Current_unit_name.is_path p then "the current compilation unit"
+        else "missing"
+      in
       fprintf ppf
-        "The module %a is an alias for module %a, which is missing"
-        !print_longident lid !print_path p
+        "The module %a is an alias for module %a, which is %s"
+        !print_longident lid !print_path p cause
```

Comment:
Minor nitpick (minor in size, major in nitpickness): I think this would be more readable if `cause` was in fact inlined at the unique use-site:
- the text source order would then match the message source order, making it easier to read
- `"the current compilation unit"` is not a "cause" (I think that if you want to keep a separate definition it should be `"is the current compilation unit"` and `"is missing"`)

Reply:
I didn't like the inlined version, so I've changed the `cause`  to include the "is ".